js/src/personal/haoking/LeftProjects.tsx

```diff
@@ -0,0 +1,29 @@
+import { Text, Title, Box } from '@mantine/core'
+import { theme } from './theme.ts'
+import styles from './Haoking.module.css'
+
+/* Component for left side of Projects page */
+export function LeftProjects() {
```

Comment:
With files like this, it may be good to start breaking up your folder into subfolders.
Would be good to have like a haoking/projects folder, and then this can just be like LeftxxxColumn() or something.

Reply:
Fixed on new commit